src/__test__/components/plots-and-tables/Volcano.test.jsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import preloadAll from 'jest-next-dynamic';
+import { Provider } from 'react-redux';
+import configureMockStore from 'redux-mock-store';
+import '@testing-library/jest-dom';
+import thunk from 'redux-thunk';
+import _ from 'lodash';
+import VolcanoPlot from '../../../pages/experiments/[experimentId]/plots-and-tables/volcano/index';
+import {
+  initialPlotConfigStates,
+} from '../../../redux/reducers/componentConfig/initialState';
+
+const mockStore = configureMockStore([thunk]);
+const experimentId = 'insaneExperiment';
+let store;
+const plotUuid = 'volcanoPlotMain';
+const dataState = {
+  cellSets: {
+    hierarchy: [
+      {
+        key: 'louvain',
+        children: [
+          {
+            key: 'louvain-0',
+          },
+          {
+            key: 'louvain-1',
+          },
+        ],
+      },
+      {
+        key: 'sample',
+        children: [
+          {
+            key: 'b7b8f6ab-88ee-4cb3-9a96-b926ef9afa5c',
+          },
+        ],
+      }],
+    properties: {
+      louvain: {
+        name: 'louvain clusters',
+        cellIds: 'Set()',
+        rootNode: true,
+        type: 'cellSets',
+      },
+      'louvain-0': {
+        name: 'Cluster 0',
+        color: '#e377c2',
+        cellIds: 'Set(1,5,14,22,24,26)',
+        rootNode: false,
+        type: 'cellSets',
+      },
+      'louvain-1': {
+        name: 'Cluster 1',
+        color: '#e377c2',
+        cellIds: 'Set(165,16,21,25,28)',
+        rootNode: false,
+        type: 'cellSets',
+      },
+      sample: {
+        name: 'Samples',
+        cellIds: 'Set()',
+        rootNode: true,
+        type: 'metadataCategorical',
+      },
+    },
+  },
+  componentConfig: {
+    [plotUuid]: { config: initialPlotConfigStates.volcano, plotType: 'volcano' },
+  },
+  differentialExpression: {
+    comparison: {
+      group: {
+        between: {
+          cellSet: null,
+          compareWith: null,
+          basis: null,
+        },
+        within: {
+          cellSet: 'louvain/louvain-0',
+          compareWith: 'louvain/rest',
+          basis: 'sample/4c762ba2-cc69-4fe5-a6b8-31ba993de86a',
```

Comment:
Where is the cell set for sample `4c762ba2-cc69-4fe5-a6b8-31ba993de86a`? Can you use a simpler sample name?

Reply:
I think you can use `all` for this